pkgs/java_http/jnigen.yaml

```diff
@@ -0,0 +1,13 @@
+summarizer:
```

Comment:
Could you add a quick comment describing how to run this e.g.

```
# Regenerate bindings with `dart run jnigen --config jnigen.yaml`.
```

Reply:
Good idea, I've added your comment verbatim as I couldn't think of any way to improve it :smiley: